examples/fetal_brain_tiny_consensus_td.mzlib.txt

```diff
@@ -0,0 +1,1807 @@
+<mzSpecLib 1.0>
+MS:1003187|library identifier=mzspec:PXL000001:fetal_brain_tiny_consensus_td
+MS:1003188|library name=fetal_brain_tiny_consensus_td
+MS:1003189|library description=A small example MS/MS consensus spectra library of 5 human fetal brain peptides, appended with 5 shuffle-and-reposition decoy spectra
+MS:1003190|library version=1.0
+MS:1001017|release date=2023-05-06
+[1]MS:1003207|library creation software=MS:1001477|SpectraST
+[1]MS:1003200|software version=6.0
+[1]MS:1003206|library creation log=COMPILE FROM /data/fetal_brain_tiny_consensus.splib"  [DECOY;c=TRUE;y=1;DPS=FALSE;I=]
+[1]MS:1003206|library creation log=> fetal_brain_tiny_consensus.splib : COMPILE FROM "/data/fetal_brain_tiny.splib"  [CONSENSUS;r=1;I=HCD;_QUO=0.6;_XNR=100;_WGT=SN;_DIS=TRUE;_XPK=150;_XPU=300;_BDN=FALSE;_TBD=FALSE]
+[1]MS:1003206|library creation log=> > fetal_brain_tiny.splib : COMPILE FROM "/data/fetal_brain.splib" ; FILTER for entries in probability table "/data/fetal_brain_tiny.txt"
+[1]MS:1003206|library creation log=> > > fetal_brain.splib : IMPORT FROM PepXML "/data/interact.pep.xml", (Comet against "UP000005640_9606.fasta" (AA); ) [P=0.9;q=9999;n=;g=FALSE;o=FALSE;I=;_RNT=0;_RDR=100000;_DCN=0;_NAA=6;_NPK=10;_MDF=99
+99;_CEN=FALSE;_XAN=FALSE;_BRK=FALSE;_BRM=FALSE;_IRT=;_IRR=FALSE]
+MS:1003203|constituent spectrum file=mzspec:PXD000561:Fetal_Brain_Gel_Velos_16_f16
+MS:1003204|constituent library file=mzspec:PXL000001:fetal_brain_tiny
+MS:1003205|constituent identification file=file://data/interact.pep.xml
+
+<AttributeSet Spectrum=all>
+MS:1000511|ms level=2
+MS:1000008|ionization type=MS:1000073|electrospray ionization
+MS:1000031|instrument model=MS:1000855|LTQ Velos
+MS:1000443|mass analyzer type=MS:1000484|orbitrap
+MS:1000044|dissociation method=MS:1000422|beam-type collision induced dissociation
+MS:1003213|mass spectrometry acquisition method=MS:1003221|data-dependent acquisition
+MS:1000465|scan polarity=MS:1000130|positive scan
+MS:1000034|intensity unit=MS:1000905|percent of base peak times 100
+[1]MS:1000045|collision energy=39.0
+[1]UO:0000000|unit=UO:0000266|electronvolt
+MS:1003072|spectrum origin type=MS:1003073|observed spectrum
+MS:1000002|sample name=human fetal brain
+[2]MS:1003254|peak attribute=MS:1003279|observation frequency of peak
```

Comment:
Peak aggregations are specified here but are absent in the peak lists themselves?

Reply:
I see now, some spectra are singletons and others are consensus, which have peak aggregations included.